bindings/kitsune_runner/src/common.rs

```diff
@@ -0,0 +1,75 @@
+use std::sync::Arc;
+
+use crate::{KitsuneAgentContext, KitsuneRunnerContext};
+use anyhow::Context;
+use kitsune_client_instrumented::WtChatter;
+use serde::{Deserialize, Serialize};
+use wind_tunnel_runner::prelude::{
+    AgentContext, HookResult, ScenarioDefinitionBuilder, WindTunnelResult,
+};
+
+#[derive(Debug, Deserialize, Serialize)]
+struct KitsuneServerUrls {
```

Comment:
Bootstrap and signal server URL are passed as CLI arguments and then combined into the connection string.